core/src/saros/session/ISarosSession.java

```diff
@@ -151,6 +151,21 @@ public void changePermission(User user, Permission permission)
    */
   public void removeListener(ISessionListener listener);
 
+  /**
+   * Enables or disables the activity execution for the given project. If the execution is disabled,
```

Comment:
A bit misleading as one might assume it works for activities applied to this project and generated by this project. Should be fine though for the current issue.

Reply:
Adjusted to better reflect that this only applies to received activities.